beamerinnerthemeOSU.sty

```diff
@@ -27,7 +27,14 @@
                 \node[text centered, text width=10cm] (author) at (\paperwidth/2, 3) {\usebeamerfont{author}\insertauthor\\ \medskip\usebeamerfont{date}\insertdate};
             \else
                 \node (graphic) at (\paperwidth/2, \paperheight/2) {\inserttitlegraphic};
-                \node[text centered, text width=14cm, osu_orange] (title) at (\paperwidth/2, 8) {\usebeamerfont{title}\inserttitle\\ \medskip\usebeamerfont{subtitle}\textcolor{osu_black}{\insertsubtitle}};
+
+		   % Compute the height of the title size
+                \newsavebox{\titlegraphicbox}
+                \savebox{\titlegraphicbox}{\inserttitlegraphic}
+                \newlength{\titlegraphicheight}
+                \settoheight{\titlegraphicheight}{\usebox{\titlegraphicbox}}
+               
+ 		   \node[text centered, text width=14cm, osu_orange] (title) at (\pagewidth/2, 3*\pageheight/4 + \titlegraphicheight/4 - 3pt) {\usebeamerfont{title}\inserttitle\\ \medskip\usebeamerfont{subtitle}\textcolor{osu_black}{\insertsubtitle}};
```

Comment:
That's great! It didn't even occur to me to use the `positioning` library. This is definitely the way to go. I think that something similar could be done to the author node, but we will save that for another time. Perhaps I'll do that sometime over the summer.